### ISSUE TITLE
Request: hidpi versions for sharp window decorations

### ISSUE THREAD

**Issue by ConnectedR** [vinceliuice/WhiteSur-kde#59]
Hello, I love the theme but I can't use the sharp versions of the window decorations because of the bug that makes window borders huge. Can you please add 1.25x/1.5x versions for those?

**Comment by ZJsheep** [comment]
I am also interested....

**Comment by rbreaves** [comment]
Indeed.. after playing around with this today I was wondering where the hidpi versions are.. pretty necessary for those that want a clean and proper rounded border look but by using other means which needs the window to remain sharp on the bottom and not doing its own poorer version of rounding things off.
